src/main/java/com/manardenza/utils/Injector.java

```diff
@@ -0,0 +1,26 @@
+package com.manardenza.utils;
+
+import com.manardenza.controller.HotelController;
+import com.manardenza.controller.UserController;
+import com.manardenza.dao.HotelDaoImpl;
+import com.manardenza.dao.UserDaoImpl;
+import com.manardenza.login.CurrentUser;
+import com.manardenza.service.HotelService;
+import com.manardenza.service.UserService;
+import lombok.Getter;
+
+import java.io.File;
+
+public class Injector {
+    private static File hotelsDatabaseFile = new File("src/main/resources/databaseFiles/databaseHotelsFile.bin");
+    private static File usersDatabaseFile = new File("src/main/resources/databaseFiles/databaseUsersFile.bin");
+    @Getter
+    private static HotelDaoImpl hotelDao = new HotelDaoImpl(hotelsDatabaseFile);
+    @Getter
+    private static UserDaoImpl userDao = new UserDaoImpl(usersDatabaseFile);
+    private static HotelService hotelService = new HotelService(hotelDao);
+    private static UserService userService = new UserService(userDao, CurrentUser.getInstance());
+    private static HotelController hotelController = new HotelController(hotelService);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "hotelController" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)